hookah.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Hookah
+
+[Service]
+ExecStart=/bin/bash -c "$GOPATH/bin/hookah --server-root $HOME/hooks --http-port 8808"
```

Comment:
As I don't wish to make Hookah specific to Go users, could we change the example binary path to `/usr/local/bin/hookah`.  My friend wasn't 100% certain $HOME would work from systemd like this? Can you validate this? I think my preference either way would be to also change the server root to maybe `/root/hooks`?

Thoughts?